management/client/client.go

```diff
@@ -97,7 +97,7 @@ func (c *Client) Sync(msgHandler func(msg *proto.SyncResponse) error) error {
 			log.Errorf("failed to open Management Service stream: %s", err)
 			return err
 		}
-
+		backOff.Reset()
```

Comment:
This won't have the desired effect if we block the connection to receive events for more than the max elapsed time

Reply:
perhaps we should move/add it to line 106

Reply:
done